app/javascript/controllers/fileManager.js

```diff
@@ -0,0 +1,410 @@
+/*
+    WebPlotDigitizer - https://automeris.io/WebPlotDigitizer
+
+    Copyright 2010-2019 Ankit Rohatgi <ankitrohatgi@hotmail.com>
+
+    This file is part of WebPlotDigitizer.
+
+    WebPlotDIgitizer is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    WebPlotDigitizer is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with WebPlotDigitizer.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+var wpd = wpd || {};
+
+wpd.utils = (function() {
```

Comment:
Is this only to be used in the `FileManager`? Should this live in a separate file?

Reply:
Ah good catch, it's actually used in `fileManager.js` and `pageManager.js` - I'll move it out to its own file

Reply:
All set